.github/workflows/gradle.yml

```diff
@@ -0,0 +1,140 @@
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
```

Comment:
Could you change the name to "Singleton-Java-Client", then we can distinguish it in Actions well?